packaging/hal-backend-ml-accelerator.spec

```diff
@@ -28,6 +28,15 @@ BuildRequires:  pkgconfig(hal-rootstrap)
 %description
 ML HAL backend drivers for various targets
 
+# Config dummy backend (dummy-passthrough)
+%define         dummy_support 1
```

Comment:
Question)
Should this option be enabled in case of DA infra?

Reply:
There could be "no actual rpm" issue in some DA infra (board that does not support NPU).
It would be better to build dummy package always and let the image maker do not choose it.